DataAccess/Public/Include/DataAccess/AbstractRepositoryContainer.hpp

```diff
@@ -4,6 +4,7 @@
 #include <memory>
 #include <stdexcept>
 #include <string>
+#include <tuple>
```

Comment:
Why not declare this header where its functionality is used? (AbstractRepositoryManager.hpp)

Reply:
I have already prepared a patch in which all these minor bugs are fixed. Waiting for Valentine's review